### PR TITLE
Add note about stale read

### PR DESCRIPTION
**Pull request by NingLin-P**
Signed-off-by: linning <linningde25@gmail.com>

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: as-of-timestamp.md [hunk 1]
```diff
@@ -32,7 +32,14 @@ TiDB 支持通过标准 SQL 接口，即通过 `AS OF TIMESTAMP` SQL 语法的
 - `AS OF TIMESTAMP TIDB_BOUNDED_STALENESS('2016-10-08 16:45:26', '2016-10-08 16:45:29')` 表示读取在 2016 年 10 月 8 日 16 点 45 分 26 秒到 29 秒的时间范围内尽可能新的数据。
 - `AS OF TIMESTAMP TIDB_BOUNDED_STALENESS(NOW() - INTERVAL 20 SECOND, NOW())` 表示读取 20 秒前到现在的时间范围内尽可能新的数据。
 
-注意： 除了指定时间戳，`AS OF TIMESTAMP` 语法最常用使用的方式是读几秒前的数据。如果采用这种方式，取值范围推荐为读 5 秒以上的历史数据。
+> **注意：**
+> 
+> 除了指定时间戳，`AS OF TIMESTAMP` 语法最常用使用的方式是读几秒前的数据。如果采用这种方式，取值范围推荐为读 5 秒以上的历史数据。
+>
+> 使用 Stale Read 时需要为 TiDB 和 PD 部署 NTP 服务，防止 TiDB 指定的时间戳超过当前最新的 TSO 分配进度（如几秒后的时间戳），或者前于 GC safe point 的时间戳。当指定的时间戳超过服务范围，TiDB 会产生报错或者等待事务提交等行为。
```
Comment (nolouch) [inline as-of-timestamp.md:39]:
```suggestion
> 使用 Stale Read 时需要为 TiDB 和 PD 节点部署 NTP 服务，防止 TiDB 指定的时间戳超过当前最新的 TSO 分配进度（如几秒后的时间戳），或者前于 GC safe point 的时间戳。当指定的时间戳超过服务范围，TiDB 会产生报错或者等待事务提交等行为。
```

**Review by nolouch — APPROVED** [review]
lgtm

**Comment by nolouch** [comment]
PTAL @TomShawn

File: as-of-timestamp.md [hunk 2]
```diff
@@ -32,7 +36,13 @@ TiDB 支持通过标准 SQL 接口，即通过 `AS OF TIMESTAMP` SQL 语法的
 - `AS OF TIMESTAMP TIDB_BOUNDED_STALENESS('2016-10-08 16:45:26', '2016-10-08 16:45:29')` 表示读取在 2016 年 10 月 8 日 16 点 45 分 26 秒到 29 秒的时间范围内尽可能新的数据。
 - `AS OF TIMESTAMP TIDB_BOUNDED_STALENESS(NOW() - INTERVAL 20 SECOND, NOW())` 表示读取 20 秒前到现在的时间范围内尽可能新的数据。
 
-注意： 除了指定时间戳，`AS OF TIMESTAMP` 语法最常用使用的方式是读几秒前的数据。如果采用这种方式，取值范围推荐为读 5 秒以上的历史数据。
+> **注意：**
+> 
+> 除了指定时间戳，`AS OF TIMESTAMP` 语法最常用使用的方式是读几秒前的数据。如果采用这种方式，取值范围推荐为读 5 秒以上的历史数据。
```
Comment (TomShawn) [inline as-of-timestamp.md:41]:
```suggestion
> 除了指定时间戳，`AS OF TIMESTAMP` 语法最常用使用的方式是读几秒前的数据。如果采用这种方式，推荐读 5 秒以上的历史数据。
```

File: as-of-timestamp.md [hunk 3]
```diff
@@ -32,7 +36,13 @@ TiDB 支持通过标准 SQL 接口，即通过 `AS OF TIMESTAMP` SQL 语法的
...
+>
+> 使用 Stale Read 时需要为 TiDB 和 PD 节点部署 NTP 服务，防止 TiDB 指定的时间戳超过当前最新的 TSO 分配进度（如几秒后的时间戳），或者前于 GC safe point 的时间戳。当指定的时间戳超过服务范围，TiDB 会产生报错或者等待事务提交等行为。
```
Comment (TomShawn) [inline as-of-timestamp.md:43]:
```suggestion
> 使用 Stale Read 时需要为 TiDB 和 PD 节点部署 NTP 服务，防止 TiDB 指定的时间戳超过当前最新的 TSO 分配进度（如几秒后的时间戳），或者超前于 GC safe point 的时间戳。当指定的时间戳超过服务范围，TiDB 会产生报错或者等待事务提交等行为。
```

File: as-of-timestamp.md [hunk 4]
```diff
@@ -32,7 +36,13 @@ TiDB 支持通过标准 SQL 接口，即通过 `AS OF TIMESTAMP` SQL 语法的
...
+>
+> `Prepare` 语句与 `AS OF TIMESTAMP` 语法的兼容支持尚不完善，不推荐同时使用
```
Comment (TomShawn) [inline as-of-timestamp.md:45]:
```suggestion
> `Prepare` 语句与 `AS OF TIMESTAMP` 语法的兼容支持尚不完善，不推荐同时使用。
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**